frontend/cypress/integration/featureFiles/overview.feature

```diff
@@ -120,9 +120,58 @@ Feature: Kiali Overview page
     Then user sees the memory chart
     And user sees the cpu chart
 
+  @multi-cluster
+  @skip
+  Scenario: The badge for local cluster should be visible
+    Then user sees the "east" cluster badge in the Kiali header 
+
+  @multi-cluster
+  @skip
+  Scenario: Namespace dropdown should not contain duplicates
+    When user opens the namespace dropdown
+    Then user should see no duplicate namespaces
+
+  @multi-cluster
+  @skip
+  Scenario: Istio panels for both clusters should be visible in the control panel
+    Then user sees the "istio-system" namespace card in cluster "east"
+    And user sees the "istio-system" namespace card in cluster "west"
+    And user sees the "Control plane" label in both "istio-system" namespace cards
+    And user sees the "Outbound policy" label in both "istio-system" namespace cards
+    And the toggle on the right side of both "istio-system" namespace cards exists
+    And the toggle should contain links to other parts of the app
+    And Istio config should not be available for the "west" "istio-system" 
+    And health should be different for "east" and "west" "istio-system"
+
   @multi-cluster
   @skip
   Scenario: See "bookinfo" in "east" and "west" clusters
     Then user sees the "bookinfo" namespace card in cluster "east"
     And user sees the "bookinfo" namespace card in cluster "west"
-    And Istio config should not be available for the "west" "bookinfo"
+    And Istio config should not be available for the "west" "bookinfo" 
+    And health should be different for "east" and "west" "bookinfo"
+
+  @multi-cluster
+  @skip
+  Scenario: The healthy status of a logical mesh application is reported in the overview of a remote cluster namespace
+    Given a healthy application in the "west" cluster
+    When I fetch the overview of the "west" cluster
```

Comment:
it seems the "$cluster" param does not exist for these functions, can you add t in function? or remove "west" from the call.